Assets/_Functionalities/OGC3DTiles/Scripts/Tile3DLayerGameObject.cs

```diff
@@ -62,14 +60,45 @@ private void EnableTileset()
         protected void Awake()
         {
             tileSet = GetComponent<Read3DTileset>();
+            
+            credentialHandler = GetComponent<ICredentialHandler>();
+            if(!string.IsNullOrEmpty(tileSet.tilesetUrl))
```

Comment:
als de tilesetUrl afwezig is, dan is dat toch error waardig en een exception? Want als BaseUri nu niet ingevuld wordt - wat gebeurd er dan verderop de pipeline omdat je toch doorgaat?

Reply:
Je kan de if weglaten - dan faalt de "new Uri" hieronder omdat het een ongeldige URI string is

Reply:
als je je eigen 3d tiles laag toevoegt `nieuwe laag (3d tiles).prefab` dan is de tilesetURL leeg totdat deze wordt ingevuld door de gebruiker (via het ui paneel). Ik denk dat deze check daarvoor is zodat je juist die exception niet krijgt als je nog bezig bent met het invullen van de gegevens.

Reply:
Ah, dan zou ik eigenlijk verwachten dat er een error state in de Layer(GameObject) geset zou worden om feedback aan de gebruiker te geven - maar we missen wel meer van dit soort feedback mechanismen, dus ik ga deze resolven